### ISSUE TITLE
[clock] Setting time format returns an error (4.6.2)

### ISSUE THREAD

**Issue by maxpowa** [embolalia/willie#713]
Brand spanking new database, I thought it might have been the old database causing issues so I created a new one.

```
$ willie --version
Willie 4.6.2 (running on python 2.7.3)
http://willie.dftba.net/
```

```
<maxpowa> .settf %F %r (%Z)
<Inumuta> ProgrammingError: Incorrect number of bindings supplied. The current statement uses 1, and there are 2 supplied. (file "/home/willie/Snakepit/local/lib/python2.7/site-packages/willie/db.py", line 685, in update)
```

Looks very similar to #697

**Comment by tyrope** [comment]
It's not only similar, it's exactly the same. Try the 5.0-RC1, or even better, a gitpull from the master branch.

**Comment by maxpowa** [comment]
Is 5.0 stable?

**Comment by tyrope** [comment]
It's less broken than 4.6's database... Whether to call it stable or not really depends on your definition of stable. It's still in Gamma, after all.

**Comment by embolalia** [comment]
This issue is resolved by the 5.0.0 release.
